lib/slack-ruby-bot/mvc/controller/base.rb

```diff
@@ -0,0 +1,111 @@
+module SlackRubyBot
+  module MVC
+    module Controller
+      class Base
+        include ActiveSupport::Callbacks
+
+        class << self
+          attr_reader :abstract
+          alias abstract? abstract
+
+          def controllers
+            Base.instance_variable_get(:@controllers)
```

Comment:
Let me get this documented and create an example to show it off first. I'll be using this regularly so as I see ways to improve it I will submit additional PRs. 

I know the `instance_variable_{set|get}` isn't as pretty as methods, but it *works*. If I figure out how to make it prettier then I'll do it, but I don't want to spend a bunch of time on something that only effects 4-5 lines of code. It isn't that important, IMHO. 

Not ignoring you... just pushing this off until later after more important stuff (i.e. docs & examples) is completed.